hack/app_sre_create_image_catalog.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -exv
+
+# prefix var with _ so we don't clober the var used during the Make build
+# it probably doesn't matter but we can change it later.
+_OPERATOR_NAME="route-monitor-operator"
+
+BRANCH_CHANNEL="$1"
+QUAY_IMAGE="$2"
+
+GIT_HASH=$(git rev-parse --short=7 HEAD)
+GIT_COMMIT_COUNT=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+
+# clone bundle repo
+SAAS_OPERATOR_DIR="saas-route-monitor-operator-bundle"
+BUNDLE_DIR="$SAAS_OPERATOR_DIR/route-monitor-operator/"
+
+rm -rf "$SAAS_OPERATOR_DIR"
+
+git clone \
+    --branch "$BRANCH_CHANNEL" \
+    https://app:"${APP_SRE_BOT_PUSH_TOKEN}"@gitlab.cee.redhat.com/service/saas-route-monitor-operator-bundle.git \
+    "$SAAS_OPERATOR_DIR"
+
+# remove any versions more recent than deployed hash
+REMOVED_VERSIONS=""
+if [[ "$REMOVE_UNDEPLOYED" == true ]]; then
+    DEPLOYED_HASH=$(
+        curl -s "https://gitlab.cee.redhat.com/service/app-interface/raw/master/data/services/osd-operators/cicd/saas/saas-${_OPERATOR_NAME}.yaml" | \
```

Comment:
Please make sure this file is going exist in app-interface or the logic won't work 😅

Reply:
I can test this code path with you if you wish

Reply:
Just make sure it exists whenever you create the saas file in app-interface